### ISSUE TITLE
Link this repo to the crates.io entry

### ISSUE THREAD

**Issue by vbatts** [sigstore/sigstore-rs#295]
**Description**

when querying https://crates.io package information, the [`sigstore` crate](https://crates.io/crates/sigstore) does not have this repo listed.

```shell
vbatts@possibly:~$ curl -sSL "https://crates.io/api/v1/crates/sigstore" | jq '.crate.repository'
null
vbatts@possibly:~$ 
```

**Comment by flavio** [comment]
This is going to be fixed by the next release of the crate

**Comment by flavio** [comment]
Fixed starting with the 0.8.0 release
